Gemfile

```diff
@@ -34,6 +34,7 @@ group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug'
 
+  gem 'rspec-rails'
```

Comment:
You already have `rspec-rails` defined below. If you move it here to the `group :development, :test` block, remove the line near the bottom of the file.